roles/edpm_multipathd/tasks/cleanup_containers.yml

```diff
@@ -0,0 +1,40 @@
+---
+# Copyright 2025 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Gather services facts
+  ansible.builtin.service_facts:
+
+- name: Identify legacy multipathd containerized services
+  ansible.builtin.set_fact:
+    legacy_multipathd_services: "{{ edpm_multipathd_legacy_services | intersect(ansible_facts.services.keys()) }}"
+
+- name: Clean up legacy multipathd containerized services
+  become: true
+  when: legacy_multipathd_services | default([]) | length > 0
+  block:
+    - name: Stop and disable legacy multipathd containerized services
+      # The edpm_tripleo_cleanup role works for stopping and disabling the
+      # legacy tripleo_multipathd.service (adoption scenario) or the
+      # edpm_multipathd.service (update scenario).
+      ansible.builtin.include_role:
+        role: osp.edpm.edpm_tripleo_cleanup
```

Comment:
We should not be using edpm_tripleo_cleanup to clean edpm services. Also AFAIR we don't clean the configs with edpm_tripleo_cleanup.

Reply:
Then I'm confused because I was asked to use that role in antoher PR. I don't know what you mean by "clean the configs" but I'm mainly interested in stopping and disabling the legacy services. I can't find the PR comments, but when I originally just used `ansible.builtin.systemd_service` to stop and disable the service I was advised to use the edpm_tripleo_cleanup role.

Reply:
Not sure who suggested that and why.. tripleo_cleanup has logic specific to tripleo services and are meant for adoption only. For container and systemd unit file cleanup there is `edpm_container_rm` role.  We also need to cleanup contaner startup configs/kolla configs generated on the host from earlier deployments that needs to be cleaned up.